src/pspm_options.m

```diff
@@ -385,6 +385,8 @@
     options = autofill(options,'dispwin',                 1,          0                 );
     options = autofill(options,'dispsmallwin',            0,          1                 );
     options = autofill(options,'fresp',                   0.5,        '>', 0            );
+    %options = autofill(options,'missing',                 [],         '*Num'            );
```

Comment:
why is this needed?

Reply:
Sorry my fault. This was used previously but no longer taking effects. I have deleted this line now.